llvm/test/CodeGen/SPIRV/basic_float_types.ll

```diff
@@ -1,12 +1,18 @@
 ; RUN: llc -O0 -mtriple=spirv-unknown-unknown %s -o - | FileCheck %s
 ; RUN: llc -O0 -mtriple=spirv32-unknown-unknown %s -o - | FileCheck %s
 ; RUN: llc -O0 -mtriple=spirv64-unknown-unknown %s -o - | FileCheck %s
+; RUN: %if spirv-tools %{ llc -O0 -mtriple=spirv-unknown-unknown %s -o - -filetype=obj | spirv-val %}
 
 define void @main() {
 entry:
+; CHECK-DAG:     %[[#half:]] = OpTypeFloat 16
```

Comment:
IS there a test checking Float16 capability is added when this type is used?

Reply:
Good catch. Added.